documentation/api/query/v4/event-counts.markdown

```diff
@@ -60,7 +61,13 @@ This endpoint builds on top of the [`events`][events] endpoint, and supports all
 ### Response format
 
 The response is a JSON array of maps. Each map contains the counts of events that matched the input
-parameters. The events are counted based on their statuses: `failures`, `successes`, `noops`, `skips`.
+parameters. The events are counted based on their statuses and corrective_change flags (PE only):
+`failures` (status = 'failure'),
+`successes` (status = 'success' and corrective_change = false),
+`noops` (status = 'noop' and corrective_change = false),
```

Comment:
No, I mean a flag within the PDB service that dispatches on the atom you asked about and either replaces the 0 counts with null in FOSS, or just removes the keys entirely. I don't think there's any point in exposing the flag to users.